### ISSUE TITLE
field can't be named `model`

### ISSUE THREAD

**Issue by he-la** [zenstackhq/zenstack#391]
**Description and expected behavior**
Create a model with a field named `model`:

```
model MyModel {
  model  String
}
```

`npx zenstack generate` errors with

```
Validation errors:
line 548: Expecting: expecting at least one iteration which starts with one of these possible Token sequences::
  <[TRIPLE_SLASH_COMMENT] ,[ID] ,[in] ,[@@]>
but found: 'model' []
line 548: Expecting: one of these possible Token sequences:
  1. [ID]
  2. [in]
but found: 'String' []
line 548: Expecting token of type '{' but found `String`. []
line 548: Expecting token of type 'EOF' but found `String`. []
```

**Environment (please complete the following information):**

-   ZenStack version: `1.0.0-alpha.113`
-   Prisma version: `4.11.0`
-   Database type: PostgreSQL

**Additional context**
Yup, I actually have a legit usecase for a field named `model`. Don't ask...

**Comment by ymc9** [comment]
I won't ask but I believe "model" is a good name for model haha. Prisma did a good job permitting it. We should fix it here.

**Comment by ymc9** [comment]
Hi @he-la , this issue is resolved by the latest alpha.114 release.

**Comment by ymc9** [comment]
Fixed by #400
